environment.yaml

```diff
@@ -3,7 +3,7 @@ channels:
   - pytorch
   - defaults
 dependencies:
-  - python>=3.9
+  - python==3.8.5
```

Comment:
Unintentional change?

Reply:
The Python version was changed in https://github.com/lstein/stable-diffusion/commit/00d2d0e90eb45bdf87a9594eeab7be730fc3eb32. Current `main` is at `3.8.5`. After that change, I've been unable to get a `development` branch running (across multiple machines, trying both 3.9.x and 3.10.x) unless I downgraded to `3.8.5` in the environment file. The changes were in relation to something @psychedelicious was doing, though I was able to run that code as well the last time I tried on 3.8.5.

Unless >=3.9 is necessary for something, or we can get it working (and probably provide users with upgrade instructions at the next release), sticking with 3.8.5 seems like a good idea at the moment.

Reply:
When I updated the YAML files, I added unversioned PIP dependencies for flask, flask-socketio and evenlet - recently added send2trash. I do not know where the versoined requirements came from; the new UI worked fine without them.

Reply:
I've never had any issues with 3.10 on Windows, including testing @psychedelicious' work